coralproject/talk
# [CORL-2818] fix mod queue ban shortcut

## What does this PR do?
This PR addresses a regression in which the `b` key shortcut no longer opened the Ban Modal in the moderation queue.

## These changes will impact:

- [ x] moderators
- [x ] admins

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags?
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As an admin or moderator navigate to the moderation queue
2. press the `b` key, and observe the the ban modal appears, and is for the user of the currently focused comment

## Where any tests migrated to React Testing Library?
No.

## How do we deploy this PR?
No special considerations needed.
